examples/scenarioOrbitManeuver.py

```diff
@@ -291,6 +300,10 @@ def run(show_plots, maneuverCase):
     scSim.ConfigureStopTime(simulationTime + T2 + T3)
     scSim.ExecuteSimulation()
 
+    # unload custom Spice kernel
+    gravFactory.unloadSpiceKernels()
+    spiceObject.unloadSpiceKernel("hst_edited.bsp", bskPath + '/supportData/EphemerisData/')
```

Comment:
can remove this line as well as we don't need to load custom kernels

Reply:
Removed line 305